src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
In the current implementation, all stars will be active by default. The stars' active state should be controlled by a modifier class, which can then be applied to the parent `.stars` block. This allows for more flexibility and reusability. Consider creating a modifier class for each possible star rating (for example, `.stars--1`, `.stars--2`, etc.), where each class modifies the nth child of `.stars__star` to display the active star image.